packages/app-store/hubspotothercalendar/api/callback.ts

```diff
@@ -0,0 +1,56 @@
+import * as hubspot from "@hubspot/api-client";
+import { TokenResponseIF } from "@hubspot/api-client/lib/codegen/oauth/models/TokenResponseIF";
+import type { NextApiRequest, NextApiResponse } from "next";
+
+import { WEBAPP_URL } from "@calcom/lib/constants";
+import prisma from "@calcom/prisma";
+
+import { decodeOAuthState } from "../../_utils/decodeOAuthState";
```

Comment:
Will this relative import work?

Reply:
I agree! I will make sure to change this in a new PR for all apps, merged this one as the autoupdate was messing merges up for me lately 😅